core/trino-main/src/main/java/io/trino/sql/planner/plan/AggregationNode.java

```diff
@@ -49,6 +51,11 @@
 {
     private final PlanNode source;
     private final Map<Symbol, Aggregation> aggregations;
+    // defines symbol for a column that selects which input should be used by final or intermediate aggregation step.
```

Comment:
> Assuming we decide per-row eventually

We fall back to per-row only in the special case where one `Page` has both raw and intermediate rows.
I've chosen null because it works without much overhead for the normal cases with partial aggregation enabled even if partitioned exchanged (or any other) does not support RLE because `ByteArrayBlockBuilder` supports
```
        if (!hasNonNullValue) {
            return new RunLengthEncodedBlock(NULL_VALUE_BLOCK, positionCount);
        }
```